docs/website/components/DocumentationDropdown.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="dropdown inline-block">
-    <button class="font-semibold py-2 px-4 rounded inline-flex items-center">
+    <button class="font-semibold py-2 pl-4 rounded inline-flex items-center">
```

Comment:
I'm not sure I understand this one. `px` will place the button evenly in the center.

Reply:
When I reworked the header structure to line up the logo with the menu, I couldn't get it fully right-aligned to line up with the rightmost menu item. This isn't a complete fix. I can go back and rework it.

Reply:
I'll redo this as a separate fix, if that's OK.